File: app/src/main/java/com/robotsandpencils/androiddaggerpracticemvp/lobby/LobbyGreetingRepository.java
package com.robotsandpencils.androiddaggerpracticemvp.lobby;

import io.reactivex.Single;

/**
 * Created by pwray on 2017-11-01.
 */

class LobbyGreetingRepository {
    Single<String> getGreeting() {
        return Single.just("Hello from LobbyGreetingRepository");
    }
}

File: app/src/main/java/com/robotsandpencils/androiddaggerpracticemvp/lobby/LobbyGreetingRepository.kt
package com.robotsandpencils.androiddaggerpracticemvp.lobby

import io.reactivex.Single

/**
 * Created by pwray on 2017-11-01.
 */

internal class LobbyGreetingRepository {
    val greeting: Single<String>
        get() = Single.just("Hello from LobbyGreetingRepository")
}
